utils/build.ps1

```diff
@@ -1316,6 +1323,7 @@ function Build-Compilers() {
       -Bin $CompilersBinaryCache `
       -InstallTo "$($Arch.ToolchainInstallRoot)\usr" `
       -Arch $Arch `
+      -AddAndroidCMakeEnv:$Android `
```

Comment:
Why the `:`?

Reply:
As far as I understand I need to make sure it's not passed in when Android is false as it's a switch flag, so this seems to be the syntax that lets me avoid passing it in when it's false correctly. When it's separate, powershell complains about passing a value to this switch flag.

Reply:
Ah, because `$Android` is empty/null rather than `$false`? I wonder if a ternary is better 🤷‍♂️

Reply:
That looks right. If using `[switch]`, the parameter doesn't take an argument and `-AddAndroidCMakeEnv` implies setting to `$true`, and `-AddAndroidCMakeEnv $booleanvalue` would imply passing `$booleanvalue` as a positional argument for a separate parameter, but the syntax also allows `-AddAndroidCMakeEnv:$booleanvalue` if you need to conditionally set it.

Otherwise one can use `[bool]`, but then you must use `-AddAndroidCMakeEnv $booleanvalue`, and just `-AddAndroidCMakeEnv` would be missing an argument.